src/expr/src/scalar/func.rs

```diff
@@ -2723,6 +2742,7 @@ where
     sqlname = "extractts",
     propagates_nulls = true
 )]
+#[allow(dead_code)]
```

Comment:
Why do we have all these dead_code markers?

Reply:
It's a work-in-progress to switch expression evaluation to these functions, but I haven't gotten around to finishing it yet.